examples/textures/01-logo-raylib.lisp

```diff
@@ -0,0 +1,28 @@
+(in-package #:claylib/examples)
+
+(defun example-textures-01 ()
+  (with-window (:title "raylib [textures] example - texture loading and drawing")
+    (let* ((image-size 256)
+           (source-rec (make-rectangle 0 0 image-size image-size +white+))
+           (dest-rec (make-rectangle (truncate (- (get-screen-width) image-size) 2)
+                                     (truncate (- (get-screen-height) image-size) 2)
+                                     image-size
+                                     image-size +white+))
+           (scene (make-scene `((texture ,(make-instance
+                                           'texture
+                                           :path "resources/raylib_logo.png"
+                                           :origin (make-vector2 0 0)
+                                           :source source-rec
+                                           :dest dest-rec
+                                           :tint +white+
+                                           :rot 0.0)))
```

Comment:
I also noticed that initializing `:rot` with an int doesn't seem to work - is that because `defwriter-float` doesn't work on the initial form?

Reply:
That would be because `texture` (unlike `rl-texture`) is missing a `definitializer`. Either I can write that or you can take a shot. It's not too hard and would probably be good to familiarize yourself with what it does if you haven't already.

Reply:
Yeah, I'd love to try. I'll get on that.